src/Model/ContextInterface.php

```diff
@@ -44,16 +44,16 @@ public function setEnabled($enabled);
     public function getEnabled();
 
     /**
-     * Set slug.
+     * Set id.
```

Comment:
Please remove that instead.

Reply:
What should I remove? The entire phpdoc block? Why?

Reply:
Just the description, because it is tautological

Reply:
See https://github.com/sonata-project/dev-kit/issues/345 and CONTRIBUTING.md

Reply:
I think @gremo just wanted to fix the copy paste mistake!